eahub/profiles/models.py

```diff
@@ -75,7 +75,7 @@ class ExpertiseArea(enum.Enum):
         GOVERNMENT_AND_POLICY: "Government and policy",
         ENTREPRENEURSHIP: "Entrepreneurship",
         SOFTWARE_ENGINEERING: "Software engineering",
-        AI_TECHNICAL_EXPERTISE: "AI technical expertise",
+        SOFTWARE_ENGINEERING_AI: "AI technical expertise",
```

Comment:
Do you want to change the label of this since you're changing the name? Or is that not the idea?